mautic/mautic
# remove Raw block from GrapesJS sidebar for MJML emails

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



Opened up [some discussion on the arguably broken state of the Raw block](https://forum.mautic.org/t/product-input-how-do-you-use-the-raw-block-in-the-email-editor/30748). My suggestion would be to remove this block entirely, since it's unusable.
It doesn't do what users think it does: renders a gallery of images instead of outputting editable html.
This seems to be intentional behavior, as [it behaves the same in GrapesJS' own demo](https://grapesjs.com/demo-mjml.html).  But since it makes no sense, we are better or removing it instead of leaving users confused.

**Note**: This PR just removes the block, it does not attempt to replace the suggested functionality. [That is a new feature in a separate PR, with its own caveats](https://github.com/mautic/mautic/pull/13386)

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Make a new MJML template based Email (eg. Paprika)
3. Notice the 'Raw' block is no longer present in the sidebar
